controllers/argocd/sso_test.go

```diff
@@ -22,8 +22,13 @@ import (
 	routev1 "github.com/openshift/api/route/v1"
 	templatev1 "github.com/openshift/api/template/v1"
 	"gotest.tools/assert"
+	k8sappsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	extv1beta1 "k8s.io/api/extensions/v1beta1"
```

Comment:
Please do not use `extensions/v1beta1` anymore, it's deprecated and got removed in K8s v1.22

Reply:
made the required changes.